modules/appeals_api/app/services/appeals_api/evidence_submission/upload_validator.rb

```diff
@@ -0,0 +1,111 @@
+# frozen_string_literal: true
+
+require 'pdf_info'
+
+module AppealsApi
+  module EvidenceSubmission
+    class UploadValidator
+      include SentryLogging
+
+      def initialize(upload)
+        @document = upload[:document]
+      end
+
+      def call
+        return file_type_error unless pdf_metadata_present?
+
```

Comment:
I think it might be possible to use guard clauses here for the entire `call` method. 

With the multi-tiered if statement, any future validations or changes might be more difficult to implement. What do you think?

Reply:
This is what I had at some point:
`return file_type_error unless pdf_metadata_present?
  return file_size_error unless valid_file_size?
  return max_dimension_error unless valid_page_dimensions?

  successful_validation_message
`
Then I just kept hearing "one return only" replaying in my head so I changed it. I prefer the return statements to the conditional. So as long as you're cool with it, i'm cool with it

Reply:
Yeah. I think that reads much more cleanly, and will be easier to adjust if necessary, personally.

Reply:
Perfect. Rewind!